README.md

```diff
@@ -1,3 +1,23 @@
 # c360-python-client
```

Comment:
Readme should includes some instructions how to use this lib.
E.g:
 - some CLIs and their usages
 - how to install this lib
 - how to contribute to this repo (since this is public repo)

Reply:
first steps addressed in https://github.com/tra-sg/c360-python-client/pull/1/commits/726b3f9d9d7cfe9a9da10d0240a3c6826ca3477b

It's not in `pypi` yet so we can't say `pip install c360-python-client`, but that's what this readme should say in the future.